kratos/utilities/mortar_utilities.h

```diff
@@ -359,6 +359,31 @@ class MortarUtilities
     }
     
     /**
+     * This function calculates the normal in a specific GP with a given shape function
+     * @param N: The shape function considered
+     * @param Geom: The geometry of condition of interest
+     */
+
+    static inline array_1d<double,3> GaussPointNormal(
```

Comment:
Call it "UnitjNormal" so it will be clear that you normalize

Reply:
OK